internal/charmstore/debug.go

```diff
@@ -15,8 +16,46 @@ func serveDebugInfo(http.Header, *http.Request) (interface{}, error) {
 	return appver.VersionInfo, nil
 }
 
-func newServiceDebugHandler() http.Handler {
+// GET /debug/check.
+func debugCheck(checks ...func() error) http.Handler {
+	return router.HandleErrors(func(http.ResponseWriter, *http.Request) error {
+		for _, check := range checks {
+			err := check()
```

Comment:
This isn't going to give us a very useful error on failure.
Also, if one is down, we don't know if both are, and we probably should.

I wonder if the checks should be map[string]func() error
and we should go through them all and if any fail
produce an error message that includes the ones that
succeeded vs failed.

Also, it would be nice to have *some* response from the
handler on success.

How about, on success:

   ["mongo ping", "es health"]

and on failure:

    some checks failed: [mongo ping: ok] [es health: connection timeout]

what do you think?